modules/currency-l1/src/main/scala/org/tessellation/currency/l1/modules/HttpApi.scala

```diff
@@ -90,6 +90,8 @@ sealed abstract class HttpApi[
   private val routes = maybeDataApplication.map { da =>
     DataApplicationRoutes[F](queues.dataApplicationPeerConsensusInput, storages.l0Cluster, da, queues.dataUpdates)
   }
+  private val dataApplicationRoutes =
+    maybeDataApplication.map(_.routes).map(r => Router("/data-application" -> r)).getOrElse(HttpRoutes.empty[F])
```

Comment:
shouldn't we wrap it by `publicRoutes`, `p2pRoutes` etc? We are exposing the endpoint details here but for other endpoints we are hiding it

Reply:
There are only public routes. What you mean by hiding the implementation, you mean the prefix?